src/pages/index.js

```diff
@@ -32,7 +32,7 @@ const App = ({ isSuccessfulSearch, searchResults, numberOfMatches, searchTerm })
                 <UniversityBody>
                     <CoronavirusNotice />
                     <GlobalNotice>
-                        This is a new service. Please help us improve it by sharing your{" "}
+                        This is a new Course Search. Please help us improve it by sharing your{" "}
```

Comment:
Minor thing - we call this app "Course Search" (with caps), but to an end user the capitalisation probably won't make much sense. Maybe "course search tool" would make more sense?
```suggestion
                        This is a new course search tool. Please help us improve it by sharing your{" "}
```